upi/openstack/bootstrap.yaml

```diff
@@ -31,14 +31,6 @@
       image: "{{ os_image_rhcos }}"
       flavor: "{{ os_flavor_master }}"
       userdata: "{{ lookup('file', os_bootstrap_ignition) | string }}"
-      auto_ip: no
+      floating_ips: "[{{ os_bootstrap_fip | default(omit) }}]"
```

Comment:
Looks like CI failure is legit:
```
TASK [Create the bootstrap server] *********************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: 'NoneType' object is not subscriptable
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/home/.ansible/tmp/ansible-tmp-1601561386.4304595-1143-222177692600218/AnsiballZ_os_server.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/tmp/home/.ansible/tmp/ansible-tmp-1601561386.4304595-1143-222177692600218/AnsiballZ_os_server.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/tmp/home/.ansible/tmp/ansible-tmp-1601561386.4304595-1143-222177692600218/AnsiballZ_os_server.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible.modules.cloud.openstack.os_server', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/lib64/python3.6/runpy.py\", line 205, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code\n    mod_name, mod_spec, pkg_name, script_name)\n  File \"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_os_server_payload_zk8lb_92/ansible_os_server_payload.zip/ansible/modules/cloud/openstack/os_server.py\", line 759, in <module>\n  File \"/tmp/ansible_os_server_payload_zk8lb_92/ansible_os_server_payload.zip/ansible/modules/cloud/openstack/os_server.py\", line 750, in main\n  File \"/tmp/ansible_os_server_payload_zk8lb_92/ansible_os_server_payload.zip/ansible/modules/cloud/openstack/os_server.py\", line 544, in _create_server\n  File \"</usr/lib/python3.6/site-packages/decorator.py:decorator-gen-4>\", line 2, in create_server\n  File \"/usr/lib/python3.6/site-packages/openstack/cloud/_utils.py\", line 378, in func_wrapper\n    return func(*args, **kwargs)\n  File \"/usr/lib/python3.6/site-packages/openstack/cloud/_compute.py\", line 957, in create_server\n    nat_destination=nat_destination,\n  File \"/usr/lib/python3.6/site-packages/openstack/cloud/_compute.py\", line 1091, in wait_for_server\n    nat_destination=nat_destination)\n  File \"/usr/lib/python3.6/site-packages/openstack/cloud/_compute.py\", line 1118, in get_active_server\n    wait=wait, timeout=timeout)\n  File \"/usr/lib/python3.6/site-packages/openstack/cloud/_floating_ip.py\", line 999, in add_ips_to_server\n    fixed_address=fixed_address)\n  File \"/usr/lib/python3.6/site-packages/openstack/cloud/_floating_ip.py\", line 912, in add_ip_list\n    fixed_address=fixed_address)\n  File \"/usr/lib/python3.6/site-packages/openstack/cloud/_floating_ip.py\", line 694, in _attach_ip_to_server\n    if not ext_ip and floating_ip['port_id']:\nTypeError: 'NoneType' object is not subscriptable\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}```

Reply:
Yes it is. auto_ip must be set to false when floating_ips is empty. But only in that case. See most recent push.